### DISCUSSION TITLE
Examples

### DISCUSSION THREAD

**Discussion by redelschaap** [spatie/laravel-livewire-wizard#12] [Q&A]
Can anyone share some visual examples of how a wizard can/would look like?

**Comment by ju5t** [comment]
You're free to design things as you like. There are no constraints. Are you looking for something specific?

**Reply by redelschaap** [reply]
No, just looking for some inspiration what can be achieved with this package.

**Reply by Programmers12** [reply]
> You're free to design things as you like. There are no constraints. Are you looking for something specific?

did anyone use this on example

**Reply by freekmurze** [reply]
I'm currently working on a simple example

**Reply by Programmers12** [reply]
> I'm currently working on a simple example

can you send me the link of the example when you done as soon

**Reply by ju5t** [reply]
https://github.com/spatie/laravel-livewire-wizard-demo-app
